app/utils/print/IndividualProtectorPrintHelper.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.print
+
+import com.google.inject.Inject
+import models.{CheckMode, NormalMode, UserAnswers}
+import pages.individual._
+import play.api.i18n.Messages
+import utils.countryOptions.CountryOptions
+import viewmodels.{AnswerRow, AnswerSection}
+
+class IndividualProtectorPrintHelper @Inject()(answerRowConverter: AnswerRowConverter,
+                                               countryOptions: CountryOptions
+                                            ) {
+
+  def apply(userAnswers: UserAnswers, provisional: Boolean, settlorName: String)(implicit messages: Messages) = {
+
+    val bound = answerRowConverter.bind(userAnswers, settlorName, countryOptions)
+
+    val add: Seq[AnswerRow] = Seq(
+        bound.nameQuestion(NamePage, "individualProtector.name", controllers.individual.routes.NameController.onPageLoad(NormalMode).url),
+        bound.yesNoQuestion(DateOfBirthYesNoPage, "individualProtector.dateOfBirthYesNo", controllers.individual.routes.DateOfBirthYesNoController.onPageLoad(NormalMode).url),
+        bound.dateQuestion(DateOfBirthPage, "individualProtector.dateOfBirth", controllers.individual.routes.DateOfBirthController.onPageLoad(NormalMode).url),
+        bound.yesNoQuestion(NationalInsuranceNumberYesNoPage, "individualProtector.nationalInsuranceNumberYesNo", controllers.individual.routes.NationalInsuranceNumberYesNoController.onPageLoad(NormalMode).url),
+        bound.ninoQuestion(NationalInsuranceNumberPage, "individualProtector.nationalInsuranceNumber", controllers.individual.routes.NationalInsuranceNumberYesNoController.onPageLoad(NormalMode).url),
```

Comment:
Should be redirecting to NationalInsuranceNumberController

Reply:
Fixed now